cms/templates/js/access-editor.underscore

```diff
@@ -4,7 +4,7 @@
         <div class="modal-section-content gating-prereq">
             <ul class="list-fields list-input">
                 <p class="field-message">
-                    <%- gettext('Select a prerequisite subsection and enter a minimum score percentage to limit access to this subsection.') %>
+                    <%- gettext('Select a prerequisite subsection and enter a minimum score percentage or minimum completion percentage to limit access to this subsection.') %>
```

Comment:
We should be explicit about the rules here. Consider changing to:
```Select a prerequisite subsection and enter a minimum score percentage and minimum completion percentage to limit access to this subsection.  Allowed values are 0-100```

Or some kind of wording so it's clear to user that both values will be used in the calculation but putting 0 for value will ignore it.